cluster-agent/go.mod

```diff
@@ -1,9 +1,10 @@
-module github.com/jgwest/managed-gitops/cluster-agent
+module github.com/redhat-appstudio/managed-gitops/cluster-agent
 
 go 1.16
 
 require (
 	github.com/argoproj/argo-cd/v2 v2.1.1
+	github.com/jgwest/managed-gitops/cluster-agent v0.0.0-20211027023712-42af25b249bf
```

Comment:
I'm guessing this line can be removed now that you have renamed from the old module name.

Reply:
Yes, totally makes sense!